### ISSUE TITLE
Challenge Designers Solving their own Challenge

### ISSUE THREAD

**Issue by Corb3nik** [OpenToAllCTF/OTA-University#71]
Should challenge designers be able to solve their own challenge?

If so, should their points count towards the leaderboard?
Should they be able to get first blood?

Suggestions so far : 
- Author can't have first blood on his own challenge
- Author's are given half the points
- No points awarded at all
- Points awarded to author after a first blood (preventing impossible challenges)
- Using another reward system like "designer coins" which can be used to purchase things (hint purchasing?)

**Comment by gellyfisher** [comment]
I agree with Author can't get first blood on his own challenge because its just not fair competition. I think challenge designers should still get full points though because after all, they were able to create and solve it with no more information than any other challenger. (except if the challenge includes guessing like forensics).

second reason is that to be able to get first place you need those points. If creating challenges is a disadvantage for myself then I won't actually create them anymore.
